viz_scripts/scaffolding.py

```diff
@@ -1,17 +1,20 @@
-import pandas as pd
+import json
+import os
+
 import numpy as np
+import pandas as pd
 
+import emission.core.get_database as edb
+import emission.core.wrapper.localdate as ecwl
```

Comment:
Do we really need to move these? The more diffs there are, the harder it is to review, and the more messed up the commit history/blame gets...

Reply:
That's fair, I have a habit of organizing/alphabetizing import statements because I took a class once where they emphasized this (https://peps.python.org/pep-0008/#imports) but I can definitely see how it adds superfluous changes.

Reply:
That is a good idea in general - but I would prefer if it could be done in a separate change that fixes imports through all the notebooks.

Reply:
Reverted these back to the way they were